.github/libs/GithubUtils.js

```diff
@@ -358,10 +279,8 @@ class GithubUtils {
                 }
 
                 issueBody += '\r\ncc @Expensify/applauseleads\r\n';
-
                 return issueBody;
             })
-            // eslint-disable-next-line no-console
             .catch(err => console.warn('Error generating comparison URL, continuing...', err));
```

Comment:
Is this warning still accurate? It sounds like this should be something like `Error generating issue body`